src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseBase.cpp

```diff
@@ -185,6 +185,8 @@ void CompositionalMultiphaseBase::postProcessInput()
                         getWrapperDataContext( viewKeyStruct::solutionChangeScalingFactorString() ) <<
                         ": The solution change scaling factor must be smaller or equal to 1.0" );
 
+  if( m_isThermal )
```

Comment:
I would add a message here if the user tries to set `useSimpleAccumulation` on a thermal simulation.

Reply:
good point but, is there a way to tell if user tries or useSimpleAccumulation=1 just came as default value?

Reply:
ok, I just used Jacques' version